src/sss_client/common.c

```diff
@@ -45,6 +45,7 @@
 #define _(STRING) dgettext (PACKAGE, STRING)
 #include "sss_cli.h"
 #include "common_private.h"
+#include "util/util_creds.h"
```

Comment:
(Just some notes, not a "change request")
Maybe makes sense to '#undef HAVE_SELINUX' here to avoid pulling <selinux/*> headers... Not sure.
Or maybe better to split "util_creds.h" into two headers... (out of this PR)